pyls/plugins/flake8_lint.py

```diff
@@ -56,6 +56,12 @@ def run_flake8(flake8_executable, args, document):
     args = [(i if not i.startswith('--ignore=') else FIX_IGNORES_RE.sub('', i))
             for i in args if i is not None]
 
+    # if executable looks like a path resolve it
+    if os.sep in flake8_executable:
```

Comment:
```suggestion
    if not os.path.isfile(flake8_executable) and os.sep in flake8_executable:
```

Let's use this only if `flake8_executable` can't be found in the filesystem.